DocContact/DocContact/DetailViewController.swift

```diff
@@ -24,8 +24,18 @@ class DetailViewController: UIViewController {
     weak var contact: Contact?
     weak var delegate: DetailViewControllerDelegate?
     
+    //variables pour afficher le profil de l'utilisateur
+    var user: User?
+    let DBManager = ManageDbProvider.sharedInstance
+    var isContactsDetails: Bool = false
```

Comment:
not clear

Reply:
Not used for the moment, it's just a base for the next step